<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 190, 255);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="wGetName" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);</string>
    </property>
    <widget class="QLabel" name="lblMakeSelection">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Make Your Selection</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>63</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnPrev">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255, 134, 255);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFavorite">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255, 134, 255);
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>â™¥</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNext">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 134, 255);</string>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="leResult">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Arial Rounded MT Bold&quot;;
border-color: rgb(170, 255, 255);
background-color: rgb(170, 255, 255);
color: rgb(255, 85, 255);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbbNumber">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255, 134, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnGo">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255, 134, 255);</string>
          </property>
          <property name="text">
           <string>GO</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblNumberWords">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 13pt &quot;Arial Rounded MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Number of words</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>lblMakeSelection</zorder>
   </widget>
   <widget class="QLabel" name="lblHeading">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;Cooper Black&quot;;
color: rgb(139, 255, 255);

</string>
    </property>
    <property name="text">
     <string>Beep Boop Beep</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescription">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>Random Name Generator</string>
    </property>
   </widget>
   <widget class="QWidget" name="wFavorites" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 85, 255);</string>
    </property>
    <widget class="QLabel" name="lblHeartedNames">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Arial Rounded MT Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Liked Names</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="lwLikedNames">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 255);
border-color: rgb(255, 255, 255);
color: rgb(255, 85, 255);
font: 12pt &quot;Arial Rounded MT Bold&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnRemove">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255, 134, 255);</string>
        </property>
        <property name="text">
         <string>   Remove   </string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
